bin/dev

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# GEM_HOME/$(echo $RUBY_IMAGE | sed -e s/:/-/g)
+
+require 'optparse'
+
+options = {}
+OptionParser.new do |opts|
+  opts.banner = 'Usage: bin/dev [options] [command]'
+
+  opts.on('-iIMAGE', '--image=IMAGE', 'Specify Docker image') do |val|
+    options[:image] = val
+  end
+end.parse!
+
+IMAGE = options.fetch(:image, 'ruby:latest')
+IMAGE_PATH_SAFE = IMAGE.gsub(':', '_')
+IMAGE_NAME = "apm-agent-ruby:#{IMAGE_PATH_SAFE}"
+VENDOR_PATH = "/vendor/#{IMAGE_PATH_SAFE}"
+
+def run(cmd)
+  system "IMAGE_NAME=#{IMAGE_NAME} #{cmd}"
+end
+
+run 'docker-compose build ' \
+  " --build-arg RUBY_IMAGE=#{IMAGE}" \
+  " --build-arg VENDOR_PATH=#{VENDOR_PATH}"
+run "docker-compose run --rm specs #{ARGV.join}"
```

Comment:
```suggestion
run "docker-compose run -u $(id -u):$(id -g) --rm specs #{ARGV.join}"
```

Reply:
👍 didn't notice this before writing the comment above